overlord/configstate/configstate.go

```diff
@@ -48,6 +48,25 @@ func ConfigureHookTimeout() time.Duration {
 
 // Configure returns a taskset to apply the given configuration patch.
 func Configure(st *state.State, snapName string, patch map[string]interface{}, flags int) *state.TaskSet {
+	summary := fmt.Sprintf(i18n.G("Run configure hook of %q snap"), snapName)
+
+	// configuration hook for "core" is handled internally
+	if snapName == "core" {
+		t := st.NewTask("configure-snapd", summary)
+		t.Set("patch", patch)
+		if flags&snapstate.UseConfigDefaults != 0 {
+			t.Set("use-defaults", true)
```

Comment:
mmh, are we missing a defaults test?

Reply:
Yes, this test was also missing, thanks for pointing it out. I added a test for this now too.